build-settings-logic/src/main/kotlin/dokkasettings.build-cache.settings.gradle.kts

```diff
@@ -1,41 +1,62 @@
 /*
  * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
-import DokkaBuildSettingsProperties.Companion.dokkaBuildSettingsProperties
 
 /**
  * Gradle Build Cache conventions.
  *
- * See [DokkaBuildSettingsProperties] for properties.
+ * Because Dokka uses Composite Builds, Build Cache must be configured consistently on
```

Comment:
minor: may be it make sense to add small note, that this file should not depend on anything (other plugins/classes from `build-settings-logic`) and so be fully independent?

Reply:
Good idea, I'll add a warning (but I guarantee it won't be read 😄)